src/core/lender/distributor/trad_distributor.sol

```diff
@@ -27,20 +27,32 @@ contract TraditionalDistributor is Distributor {
     // ERC20
     CurrencyLike public currency;
 
-    function file(bytes32 what, address addr) public {
+    constructor(address shelf_) Distributor(shelf_)  public {
+        poolClosing = false;
```

Comment:
This is the only place in the codebase where we call this a pool. Maybe we should call it something else?

Reply:
good point